src/Chainweb/Utils.hs

```diff
@@ -188,45 +190,47 @@ import Control.Monad.IO.Class
 import Control.Monad.Reader as Reader
 
 import Data.Aeson.Text (encodeToLazyText)
-import qualified Data.Aeson.Types as Aeson
-import qualified Data.Attoparsec.Text as A
 import Data.Bifunctor
 import Data.Bits
+import Data.ByteString (ByteString)
 import Data.Bytes.Get
 import Data.Bytes.Put
-import Data.ByteString (ByteString)
-import qualified Data.ByteString as B
-import qualified Data.ByteString.Base64 as B64
-import qualified Data.ByteString.Base64.URL as B64U
-import qualified Data.ByteString.Lazy as BL
 import Data.Either (partitionEithers)
 import Data.Foldable
 import Data.Functor.Of
 import Data.Hashable
-import qualified Data.HashMap.Strict as HM
-import qualified Data.HashSet as HS
 import Data.List.NonEmpty (NonEmpty(..))
-import qualified Data.List.NonEmpty as NEL
 import Data.Monoid (Endo)
 import Data.Proxy
 import Data.Serialize.Get (Get)
 import Data.Serialize.Put (Put)
 import Data.String (IsString(..))
-import qualified Data.Text as T
-import qualified Data.Text.Encoding as T
-import qualified Data.Text.Lazy as TL
+import Data.Text (Text)
 import Data.These (These(..))
 import Data.Tuple.Strict
 import Data.Word (Word64)
-
 import GHC.Generics
 import GHC.Stack (HasCallStack)
 import GHC.TypeLits (KnownSymbol, symbolVal)
+import qualified Data.Aeson.Types as Aeson
+import qualified Data.Attoparsec.Text as A
+import qualified Data.ByteString as B
+import qualified Data.ByteString.Base64 as B64
+import qualified Data.ByteString.Base64.URL as B64U
+import qualified Data.ByteString.Lazy as BL
+import qualified Data.HashMap.Strict as HM
+import qualified Data.HashSet as HS
+import qualified Data.List.NonEmpty as NEL
+import qualified Data.Text as T
+import qualified Data.Text.Encoding as T
+import qualified Data.Text.Lazy as TL
```

Comment:
The imports in chainweb are usually sorted by module name (not by line). Could you please revert this reordering. It adds a lot of noise to the PR.

Reply:
It must have been a slip of the finger. I'll revert this.